### PR TITLE
Minor edits to FAST network stage READMEs

### LINKED ISSUE #1699: Update  READMEs of nva stages
The READMEs for the two nva stages (2-networking-c-nva, 2-networking-e-nva-bgp) are out-of-date wrt to what they do.

A few examples of things that are no longer true:
- PSA ranges created by default
- Proxy only subnets created by default
- Some ranges announced to onprem

and probably some more.

### PR DESCRIPTION
**Pull request by ludoo**
Fixes #1699

### CODE REVIEW

File: fast/stages/2-networking-d-separate-envs/README.md [hunk 1]
```diff
@@ -112,15 +112,12 @@ This configuration is battle-tested, and flexible enough to lend itself to simpl
 ### VPCs
 
 VPCs are defined in separate files, one for each of `prod` and `dev`.
-Each file contains the same resources, described in the following paragraphs.
 
-The **project** ([`project`](../../../modules/project)) contains the VPC, and enables the required APIs and sets itself as a "[host project](https://cloud.google.com/vpc/docs/shared-vpc)".
+These files contain different resources:
```
Comment (juliocc) [inline fast/stages/2-networking-d-separate-envs/README.md:116]: which files?

Seems like this header doesn't match what is enumerated later

File: fast/stages/2-networking-d-separate-envs/README.md [hunk 2]
```diff
@@ -112,15 +112,12 @@ This configuration is battle-tested, and flexible enough to lend itself to simpl
...
-
-**Cloud NAT** ([`net-cloudnat`](../../../modules/net-cloudnat)) manages the networking infrastructure required to enable internet egress.
+- **project** ([`project`](../../../modules/project)) contains the VPC, and enables the required APIs and sets itself as a "[host project](https://cloud.google.com/vpc/docs/shared-vpc)".
+- **VPCs** ([`net-vpc`](../../../modules/net-vpc)): manage the subnets, the explicit routes for `{private,restricted}.googleapis.com` and the DNS inbound policy (for the trusted landing VPC). Non-infrastructural subnets are created leveraging "resource factories": the configuration is separated from the module that implements it, and stored in a well-structured file. To add a new subnet, simply create a new file in the `data_folder` directory defined in the module, following the examples found in the [Fabric `net-vpc` documentation](../../../modules/net-vpc#subnet-factory). Sample subnets are shipped in [data/subnets](./data/subnets) and can be easily customized to fit users' needs. [PSA](https://cloud.google.com/vpc/docs/configure-private-services-access#allocating-range) are configured by the variable `psa_ranges` if managed services are needed.
```
Comment (juliocc) [inline fast/stages/2-networking-d-separate-envs/README.md:119]: - s/manage/manages/?
- '(for the trusted landing VPC)' -> 'for the trusted landing VPC'
- '"resource factories"' -> 'resource factories' (why the quotes?)
- I'd remove everything between the semicolon and and "Sample subnets are shipped..."

**Review by juliocc — APPROVED** [review]

File: fast/stages/2-networking-d-separate-envs/README.md (hunk 1 above)
Comment (ludoo) [reply fast/stages/2-networking-d-separate-envs/README.md:116]: image

`defined in separate files`...

File: fast/stages/2-networking-d-separate-envs/README.md (hunk 2 above)
Comment (ludoo) [reply fast/stages/2-networking-d-separate-envs/README.md:119]: Just open the PR and do your edits Julio, it will be so much quicker :)

Comment (ludoo) [reply fast/stages/2-networking-d-separate-envs/README.md:119]: BTW these were all already in the file. :)

**Pull request merged**